RELEASE-NOTES.md

```diff
@@ -9,19 +9,17 @@ Packages released:
 - @patternfly/patternfly@v2.71.1
 
 ### Components
-- **File upload:**
-  - Removed message container, added form to error example ([#2807](https://github.com/patternfly/patternfly/pull/2807)
-- **Table:**
-  - Updated text in column management demo modal ([#2875](https://github.com/patternfly/patternfly/pull/2875)
+- **File upload:** Removed message container, added form to error example ([#2807](https://github.com/patternfly/patternfly/pull/2807)
```

Comment:
need to add  a closing parenthesis to all of the PR links

```suggestion
- **File upload:** Removed message container, added form to error example ([#2807](https://github.com/patternfly/patternfly/pull/2807))
```

Reply:
Fixed!